proto/folio/taxed_incidental_reservation_nightly_quote.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package holms.types.folio;
+option csharp_namespace = "HOLMS.Types.Folio";
+option go_package = "folio";
+
+import "primitive/monetary_amount.proto";
+import "folio/tax_fee_quote.proto";
+import "primitive/pb_local_date.proto";
+import "supply/incidental_items/incidental_item_indicator.proto";
+
+message TaxedIncidentalReservationNightlyQuote {
+	holms.types.primitive.PbLocalDate opsdate = 1;
+	holms.types.primitive.MonetaryAmount gross_incidental_charge = 2;
```

Comment:
What is the context in which this is "gross"? Is it the amount before tax/assessment?

Usually "Gross" is only used in contexts where adjustments are subtractive, because it's larger but gets smaller, as the name implies.